### DISCUSSION TITLE
Entries missing after import from Keepass

### DISCUSSION THREAD

**Discussion by HolGORE** [dani-garcia/vaultwarden#3406] [Q&A] [answered]
Hello all,

i wasn't able to find an answer, so i will try it here.

I set up vaultwarden yesterday, everything like webui, access, etc. works.

Now i tried to import my data from keepass (using the XML v2 export in keepass).
Title, user name, password is set at least for each entry.

but now if i import the XML in vaultwarden some entries are missing. i already found 2, which are missing, not sure how many more.
in the XML file the missing entries are available.

Anyone else faced the same issue?

thanks!

**Comment by HolGORE** [comment, marked as answer]
I found the problem:
in the vaultwarden/data/bitwarden.log i found an error
[ERROR] The field Notes exceeds the maximum encrypted value length of 10000 characters.

**Reply by BlackDex** [reply]
Which version of Vaultwarden are you running? Since you should receive a message regarding this, and import should fail fully instead of skipping those.

**Reply by HolGORE** [reply]
> Which version of Vaultwarden are you running? Since you should receive a message regarding this, and import should fail fully instead of skipping those.

i just started using vultwarden last week, installed it on 31st march.
i didn't get a notification or anything. only found the error in the logs and some of the entries has been imported.

**Reply by BlackDex** [reply]
That doesn't tell me which version it is.
Could you provide the version used? So that we can try and replicate it somehow. Since again, this should have been solved already.
